content/blog/survey-02/index.md

```diff
@@ -0,0 +1,221 @@
+More pessimistically, this means nearly half of the readers would change something about the content quantity. 
+17% voted for "less content; keep only the most important news" and 25% for "more content; add sections for minor news".
+These two options are luckily not mutually exclusive. 
+One option we could implement is to have a new section for "minor news" where we don't go into detail, 
+and a section for "miscellaneous links" where we only list some links without any commentary.
+
+## Newsletter Frequency
+
+![Newsletter frequency barplot](frequency.png)
+
+73% of readers are either happy with the current frequency or don't care. A minority of 21% would like the newsletter to become quarterly. 
+Arguments we've heard for this are that a lower frequency would allow editors to improve the quality that goes into each newsletter. 
+Counterarguments include that a lower frequency would make the newsletter less timely. 
+Things like calls for playtesters or job offers would be less useful if they were only sent out every three months.
+
+## AI
+
+![AI barplot](ai.png)
+
+This question was a catalyst for a lot of discussion on [Discord]. 
+
+If we interpret the answers as a scale of 1-5, where 1 is "not okay at all" and 5 is "I love it", the mean answer was 2.25, the median 2.
+Notably, the mode is tied at 1 and 2. The 95% confidence interval is [1.89, 2.61] using a standard error of 0.18. 
+People are generally against using an LLM to generate summaries
+(the p-value of a t-test of mean 3 is 1.2e-4).
+
+While a majority of readers (65%) are at least okay with AI-generated summaries, a significant minority (35%) are not okay at all with this proposal. 
+These include very active members of the community and 
+contributors who have announced that they would no longer want their content to be included in the newsletter if AI was used.
+
+Of note is that the verbal feedback we got indicated that a lot of readers did not fully understand what exactly was being proposed. 
+People worried that we would start generating a majority of the newsletter or entire sections with AI, which is not something any of us wants. 
+Some readers also thought we already started using LLMs.
+The actual idea was to use AI to generate summaries of articles that were already hand-picked by the editors but not summarized yet because of time constraints.
+The summaries would then be edited and verified by the editors. The extent to which AI would be used would be limited to up to two sentences per late article.
+Any confusion in this regard is our fault.
+We will try to be more clear on such questions in the future.
+
+Although this misunderstanding might have skewed the results, we have reason to believe that the effect is not too large. 
+Anecdotally, when we properly explained the proposal to readers who were against it, 
+most did not change their mind and cited deontological reasons for their stance. 
```

Comment:
deontological is a big, relatively unclear word here.

Reply:
fair enough, haha